recipes/blastalign/meta.yaml

```diff
@@ -0,0 +1,32 @@
+package:
+  name: blastalign
+  version: "1.4"
+    
+build:
+  number: 0
+  skip: True # [osx]
+    
+source:
+  fn: BlastAlign.tar
+  url: https://sites.google.com/site/belshawrobert/software/BlastAlign.tar?attredirects=0&d=1
```

Comment:
Okay, I'm trying right now. I'm not used to Git yet and I realize I created another pull request instead of continue this one, is that okay ?